.github/container/install-cudnn.sh

```diff
@@ -7,21 +7,24 @@ export TZ=America/Los_Angeles
 
 apt-get update
 
-# Extract CUDA version from `nvcc --version` output line
+# Extract major CUDA version from `nvcc --version` output line
 # Input: "Cuda compilation tools, release X.Y, VX.Y.Z"
-# Output: X.Y
-cuda_version=$(nvcc --version | sed -n 's/^.*release \([0-9]*\.[0-9]*\).*$/\1/p')
+# Output: X
+cuda_major_version=$(nvcc --version | sed -n 's/^.*release \([0-9]*\.[0-9]*\).*$/\1/p' | cut -d. -f1)
 
 # Find latest cuDNN version compatible with existing CUDA by matching
-# ${cuda_version} in the package version string
-libcudnn_version=$(apt-cache show libcudnn8 | sed -n "s/^Version: \(.*+cuda${cuda_version}\)$/\1/p" | head -n 1)
-libcudnn_dev_version=$(apt-cache show libcudnn8-dev | sed -n "s/^Version: \(.*+cuda${cuda_version}\)$/\1/p" | head -n 1)
+# ${cuda_major_version} in the package version string
+# In most cases cuDNN release is behind CUDA ones. It is considered, that major 
```

Comment:
most -> some.
I'm not sure

Reply:
Here is cites from local build log:
```
<...>
#4 [ 1/15] FROM docker.io/nvidia/cuda:12.3.0-devel-ubuntu22.04@sha256:10aee9352d37507f233317d19feb6f39ad3d909189fe56c8b3d56ec27ab12204
#4 CACHED
<...>
#12 9.016 + apt-get install -y libcudnn8=8.9.6.50-1+cuda12.2 libcudnn8-dev=8.9.6.50-1+cuda12.2
#12 9.033 Reading package lists...
#12 9.388 Building dependency tree...
#12 9.463 Reading state information...
#12 9.543 The following NEW packages will be installed:
#12 9.543   libcudnn8 libcudnn8-dev
#12 9.703 0 upgraded, 2 newly installed, 0 to remove and 65 not upgraded.
#12 9.703 Need to get 878 MB of archives.
#12 9.703 After this operation, 2354 MB of additional disk space will be used.
#12 9.703 Get:1 https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/sbsa  libcudnn8 8.9.6.50-1+cuda12.2 [442 MB]
#12 132.0 Get:2 https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/sbsa  libcudnn8-dev 8.9.6.50-1+cuda12.2 [436 MB]
#12 196.8 debconf: delaying package configuration, since apt-utils is not installed
#12 196.9 Fetched 878 MB in 3min 7s (4692 kB/s)
#12 196.9 Selecting previously unselected package libcudnn8.^M
#12 196.9 (Reading database ... ^M(Reading database ... 5%^M(Reading database ... 10%^M(Reading database ... 15%^M(Reading database ... 20%^M(Reading database ... 25%^M(Reading database ... 30%^M(Reading database ... 35%^M(Reading database ... 40%^M(Reading database ... 45%^M(Reading database ... 50%^M(Reading database ... 55%^M(Reading database ... 60%^M(Reading database ... 65%^M(Reading database ... 70%^M(Reading database ... 75%^M(Reading database ... 80%^M(Reading database ... 85%^M(Reading database ... 90%^M(Reading database ... 95%^M(Reading database ... 100%^M(Reading database ... 27481 files and directories currently installed.)^M
#12 196.9 Preparing to unpack .../libcudnn8_8.9.6.50-1+cuda12.2_arm64.deb ...^M
#12 196.9 Unpacking libcudnn8 (8.9.6.50-1+cuda12.2) ...^M
#12 220.1 Selecting previously unselected package libcudnn8-dev.^M
#12 220.1 Preparing to unpack .../libcudnn8-dev_8.9.6.50-1+cuda12.2_arm64.deb ...^M
#12 220.1 Unpacking libcudnn8-dev (8.9.6.50-1+cuda12.2) ...^M
#12 243.4 Setting up libcudnn8 (8.9.6.50-1+cuda12.2) ...^M
#12 243.4 Setting up libcudnn8-dev (8.9.6.50-1+cuda12.2) ...^M
#12 243.4 update-alternatives: using /usr/include/aarch64-linux-gnu/cudnn_v8.h to provide /usr/include/cudnn.h (libcudnn) in auto mode^M
#12 243.4 + apt-get clean
<...>
#12 DONE 234.5 s
```

Base build is successful. How to check that it works? Is running `test_jax.sh` enough?

Reply:
Done. Run `tests/lax_test.py` successfully.